### DISCUSSION TITLE
[Feature Request] Support granular permissions for all Package registries

### DISCUSSION THREAD

**Discussion by GSala** [community/community#19829] [Actions]
Repository-scoped permissions are not enough for access control within an organization that shares packages with external parties.

**Comment by themiralay** [comment]
[This present issue](https://github.com/github-community/community/discussions/7575) is probably most popular issue with regard to tasks in Github Projects (Beta)!
 
 Other (possible) duplicates are:
 
 * [[Projects Beta] Show number of tasks vs completed task after issue title #16616](https://github.com/orgs/github-community/discussions/16616)
 * [[Projects Beta] Show x of n tasks on issue #13033](https://github.com/orgs/github-community/discussions/13033)
 * [[Projects beta] Task list progress #16721](https://github.com/orgs/github-community/discussions/16721)
 * [Show percent tasks complete in a new projects Beta field #12050](https://github.com/orgs/github-community/discussions/12050)
